packages/base/src/util/dragAndDrop/handleDragOver.ts

```diff
@@ -0,0 +1,61 @@
+import type UI5Element from "../../UI5Element.js";
+import type MovePlacement from "../../types/MovePlacement.js";
+import DragRegistry from "./DragRegistry.js";
+
+type DragOverResult = {
+	targetReference: HTMLElement | null;
+	placement: any;
+}
+
+type DragPosition = {
+	element: HTMLElement;
+	placements: MovePlacement[];
+};
+
+/**
+ * Handles the dragover event.
+ */
+function handleDragOver(e: DragEvent, component: UI5Element, position: DragPosition, target: HTMLElement): DragOverResult {
+	const draggedElement = DragRegistry.getDraggedElement();
+	const dragOverResult: DragOverResult = {
+		targetReference: null,
+		placement: null,
+	};
+
+	if (!draggedElement) {
+		return dragOverResult;
+	}
+
+	const placements = position.placements;
+	dragOverResult.targetReference = e.target as HTMLElement;
+
+	const placementAccepted = placements.some(placement => {
+		const beforeItemMovePrevented = !component.fireDecoratorEvent("move-over", {
+			originalEvent: e,
+			source: {
+				element: draggedElement,
+			},
+			destination: {
+				element: target,
+				placement,
+			},
+		});
+
+		if (beforeItemMovePrevented) {
+			e.preventDefault();
```

Comment:
how this work if two events are registered and one allows dragnand the other not?